static/src/javascripts/bootstraps/enhanced/atoms.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import config from 'lib/config';
+import { services } from 'projects/atoms/services';
+
+const bootstrapAtom = <A>(atomMaker: AtomMaker<A>, atomType: AtomType) => {
+    const atomBuilder = atomMaker[atomType].default(services);
+    [
+        ...document.querySelectorAll(`[data-atom-type='${atomType}']`),
+    ].forEach(atomDom => {
+        const atom = atomBuilder(atomDom).runTry();
+        if (typeof atom === 'string') {
+            // eslint-disable-next-line no-console
+            console.log(
+                `Failed to initialise atom [${atomType}/${atomDom.getAttribute(
+                    'data-atom-id'
+                ) || ''}]: ${atom}`
+            );
+        } else if ('requestIdleCallback' in window) {
+            requestIdleCallback(() => {
+                atom.start();
+            });
+        } else {
+            atom.start();
+        }
+    });
+};
+
+const initAtoms = () => {
+    if (config.get('page.atomTypes.guide')) {
+        require.ensure(
+            ['@guardian/atom-renderer/dist/guide/article/index.css'],
+            require => {
+                require('@guardian/atom-renderer/dist/guide/article/index.css');
```

Comment:
if these are `require`d here, do they need to appear in array above? i think this statement should get them in?

Reply:
That's a good question, I have no idea. Will try.

Reply:
Works! I got confused by the documentation 😆